delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Projects.scala

```diff
@@ -257,4 +262,16 @@ object Projects {
       case c: DeprecateProject => deprecate(c)
     }
   }
+
+  /**
+    * Create an instance of [[EventExchange]] for [[ProjectEvent]].
+    * @param projects  projects operation bundle
+    */
+  def eventExchange(projects: Projects)(implicit resolution: RemoteContextResolution): EventExchange =
```

Comment:
I thought we were waiting before indexing project events ?

Reply:
No. Otherwise things crash now. We are doing it and we are signaling the result as expected.